packages/commonwealth/client/scripts/views/pages/discussions/HeaderWithFilters/HeaderWithFilters.tsx

```diff
@@ -199,13 +216,26 @@ export const HeaderWithFilters = ({
   return (
     <div className="HeaderWithFilters">
       <div className="header-row">
-        <CWText type="h3" fontWeight="semiBold" className="header-text">
-          {isUndefined(topic)
-            ? isOnArchivePage
-              ? 'Archived'
-              : 'All Discussions'
-            : topic}
-        </CWText>
+        {showTabs && tabs && tabs.length ? (
+          <CWTabsRow>
+            {tabs.map((tab, index) => (
+              <CWTab
+                key={index}
+                label={tab.label}
+                onClick={() => updateActiveTab?.(tab.value)}
+                isSelected={selectedTab === tab.value}
+              />
+            ))}
+          </CWTabsRow>
+        ) : (
+          <CWText type="h3" fontWeight="semiBold" className="header-text">
+            {isUndefined(topic)
+              ? isOnArchivePage
+                ? 'Archived'
+                : 'All Discussions'
+              : topic}
+          </CWText>
+        )}
```

Comment:
lets store the active tab in url as we do elsewhere. ex: visiting `/discussions?tab=all` should take user to all discussions and `/discussions?tab=overview` should take to overview + lets make sure the URL updates sync with the filters in discussion section